src/main/webapp/app/exercises/shared/participation-submission/participation-submission.component.html

```diff
@@ -72,6 +72,20 @@ <h4 *ngIf="exercise?.assessmentDueDate">
                             [showGradedBadge]="!isTmpOrSolutionProgrParticipation"
                             [showTestDetails]="isTmpOrSolutionProgrParticipation"
                         ></jhi-result>
+                        <span
+                            class="badge"
+                            style="border-radius: 0px"
+                            jhiDeleteButton
+                            [buttonSize]="buttonSizeSmall"
+                            [actionType]="actionTypeEmpty"
+                            [entityTitle]="'deleteResultButton'"
+                            deleteQuestion="artemisApp.result.delete.question2"
+                            (delete)="deleteResult(result.submission, result)"
+                            [dialogError]="dialogError$"
+                        >
+                            <fa-icon [icon]="'times'"></fa-icon>
+                            {{ 'artemisApp.result.delete.buttonText' | artemisTranslate | uppercase }}
+                        </span>
```

Comment:
why is this outside of the `jhi-result` whereas the GRADED badge is inside this component. IMO the deletion of a result should be placed in the `jhi-result` component as well. 
Also, why are you using the inline style `border-radius: 0px`? This leads to different styles directly shown next to each other:
![image](https://user-images.githubusercontent.com/41366522/119717658-dd14c680-be66-11eb-8aca-70ae7379d326.png)

Reply:
So, the different style is intended so it is clear that it is a button, and not just a badge. 

When I started to implement it, I actually thought about putting the delete button in the `jhi-result` as well, but decided against it. `jhi-result` is used in a lot of places, and I only want to show this button on this one page. So I think it makes more sense if the delete functionality is not inside the `jhi-result` itself, as it won't be used almost all of the time. (It keeps the `jhi-result` a bit simpler)

Reply:
aaah true! makes perfect sense. maybe somewhat more space between the two of them would underline the different functionality badge vs. button... or not writing uppercased DELETE buttons are not uppercased (such as the delete submission on the right) but badges are? does that make sense?

Reply:
I agree that it should be clear that its a button. Not using uppercase letters would be useful here to make the functionality more clear :)

Reply:
In the end I would say an active verb like "delete" implies that there is the possibility to do an action with this button instead if the adjective "graded" for the badge. Not sure if every use gets that, though :D Not using all uppercase letter might clarify this even more

Reply:
I removed the uppercase pipe :+1: